webpack.config.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
     contentBase: './',
   },
   resolve: {
```

Comment:
I think you can remove this entirely. The [defaults handle this](https://webpack.js.org/configuration/resolve/).